### ISSUE TITLE
Library Still Supported?

### ISSUE THREAD

**Issue by jeffyamada** [payeezy/payeezy_direct_API#32]
It looks like this package hasn't been touched or updated in years. I'm wondering if anyone else out there is trying to determine if [these updates](https://developer.payeezy.com/content/upcoming-changes-rest-api-endpoints) will affect transactions using this library?

**Comment by UdyW** [comment]
Looks like the library not supported anymore. Its disappointing given that its maintained by FirstData

**Comment by jeffyamada** [comment]
@UdyW Yeah, pretty odd, and I ended up speaking with them directly. They were somewhat unfamiliar with this package, but did confirm that it should keep working. It's now on our roadmap to transition away from this library though.

**Comment by UdyW** [comment]
@jeffyamada What library would you suggest? I have tried Omnipay First Data drive its even worst than this one.

**Comment by burtontanner** [comment]
Its still works for me when using version 1 of paymentJS. I'm not sure about version 2 though.

**Comment by morenoh149** [comment]
yep that's the thing. Even if these examples are outdated. The api's have not changed. Apis are a contract and should not change often. As long as you have basic http message descriptions, or working curl requests, you should be able to craft the http requests using whatever stack you want.

**Comment by burtontanner** [comment]
I just got it to work with version2 of payment js. One note is that you need to convert the new token JSON format to the old format. 
``` javascript 

let v2TokenFormat = {
	"bin": "123123",
	"brand": "visa",
	"exp": {"month": "01", "year": "2030"},
	"last4": "1111",
	"masked": "XXXXXXXXXXXX1111",
	"name": "Bob Sagget",
	"token": "1234123412341234"
};

let v1TokenFormat = {"type": "visa", "cardholder_name": "Bob Sagget", "exp_date": "1030", "value": "1234123412341234"};

let convertToPaymentJs1Format = (tokenCard) => {
	if (tokenCard.type) return tokenCard;
	let oldFormat = {};
	oldFormat.type = tokenCard.brand;
	oldFormat.cardholder_name = tokenCard.name;
	oldFormat.value = tokenCard.token;
	oldFormat.token = tokenCard.token;
	oldFormat.exp_date = tokenCard.exp.month + tokenCard.exp.year.substr(2, 2);
	return oldFormat;
};
 ```
